backend/core/archer.ts

```diff
@@ -320,6 +320,7 @@ export const ArcherListing: Listing = {
     createdAt: new Date(),
     updatedAt: new Date(),
     city: "San Jose",
+    county: "San Jose",
```

Comment:
jw where do we use this file?

Reply:
I have no idea, but figured it would need county if it's ever needed.

Reply:
@ifranch good catch! I updated to Santa Clara.